packages/__docs__/src/Nav/props.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 - present Instructure, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import type { PropValidators } from '@instructure/shared-types'
+import PropTypes from 'prop-types'
+type NavOwnProps = {
+  docs: Record<string, any>
+  sections: Record<string, any>
+  themes?: Record<string, any>
+  icons?: Record<string, any>
+  selected?: string
```

Comment:
Actually these could be typed a bit more precisely. As far as I can tell from the code, the Nav gets its props from the `docs-data.json` (it is in the `packages/__docs__/__build__` folder). If you copy the code from the json to a json formatter (e.g.: https://codebeautify.org/jsonviewer) then you can browse the tree easily and you can see how the data looks like. Question is: is it worth typing this json? Let's talk about it on the stand-up!